src/wasm/wasm.cpp

```diff
@@ -796,7 +796,10 @@ void MemoryGrow::finalize() {
   }
 }
 
-void RefNull::finalize(HeapType heapType) { type = Type(heapType, Nullable); }
+void RefNull::finalize(HeapType heapType) {
+  assert(heapType.isBottom());
+  type = Type(heapType, Nullable);
```

Comment:
```suggestion
  type = Type(heapType, Nullable);
  assert(type.isNull());
```
(maybe excessive though?)

Reply:
Yeah, this seems excessive since we already checked that `heapType.isBottom()`.